src/client/common.py

```diff
@@ -1,19 +1,16 @@
-import base64
-import time
-import hmac
-import os
-import hashlib
-import six
-import sys
+import base64, time, hmac, os, hashlib
+import six, sys
+from six.moves import urllib
 
 import json
 import pprint
 import requests
-from six.moves import urllib
 import urllib as ul
 import xmltodict
 
 import exceptions
+import binascii, getpass
+from Crypto.PublicKey import RSA
```

Comment:
Have we ensured this library would always be present in the system? And this doesnt have version dependencies.